bootstrap/eks/api/v1alpha3/eksconfigtemplate_types.go

```diff
@@ -42,7 +49,8 @@ type EKSConfigTemplate struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec EKSConfigTemplateSpec `json:"spec,omitempty"`
+	Spec   EKSConfigTemplateSpec   `json:"spec,omitempty"`
+	Status EKSConfigTemplateStatus `json:"status,omitempty"`
```

Comment:
Hmm, this line shouldn't be here. EKSConfigTemplate has no status. Something go wrong with a rebase?